tests/sniffer.py

```diff
@@ -23,7 +23,7 @@ def start(self, outputfile_basename):
         debug("Starting tcpdump, output file {}.pcap".format(outputfile_basename))
         os.makedirs(os.path.join(self.tcpdump_log_dir, 'logs'), exist_ok=True)
         outputfile = os.path.join(self.tcpdump_log_dir, outputfile_basename) + ".pcap"
-        command = ["tcpdump", "-i", self.interface, "-w", outputfile]
+        command = ["tcpdump", "-i", self.interface, '-U', '-w', outputfile]
```

Comment:
Maybe it would be worth adding the `-s` option with the maximum size of a wireless frame (2346).
The default size depends on the tcpdump version, but on my machine for example it's set to 262144, while we know the maximum size we'll ever capture can't exceed 2346 (and probably even 1516 in practice).

AFAIU tcpdump sets it so high for backward compatibility, but they recommend to set it to something smaller (from man tcpdump):

> You should limit snaplen to the smallest number that will  capture the protocol information you're interested in.

(it's a nitpick comment of course, we shouldn't have that much traffic for now, so we can still do that once we start sniffing wireless frames)

Reply:
I believe snaplen only has an impact on memory usage, not so much on performance. So I'm not really worried about it. That said, it's a good suggestion.

BTW I intend to also drop tcpdump and replace it with tshark at some point - it's kind of stupid to require both to be installed.